pom.xml

```diff
@@ -58,9 +58,9 @@ limitations under the License.
         <hamcrest.version>1.3</hamcrest.version>
 
         <!-- google-cloud-java related dependency versions -->
-        <google-cloud.version>0.97.0-alpha</google-cloud.version>
+        <google-cloud.version>0.102.0-alpha</google-cloud.version>
         <opencensus.version>0.21.0</opencensus.version>
-        <guava.version>27.1-android</guava.version>
+        <guava.version>28.0-android</guava.version>
```

Comment:
Do we still need to specify this? I would think that we inherit it from google-cloud-clients

Reply:
Unfortunately yes, We are referring this version in [bigtable-beam-import](https://github.com/googleapis/cloud-bigtable-client/blob/master/bigtable-dataflow-parent/bigtable-beam-import/pom.xml#L48) & [bigtable-emulator](https://github.com/googleapis/cloud-bigtable-client/blob/master/bigtable-test/bigtable-emulator-maven-plugin/pom.xml#L52).

Reply:
Ok, can you add a comment to avoid future confusion?

Reply:
Added comment and grouped guava version with the beam.